client-data/tools/pencil/pencil.js

```diff
@@ -153,11 +171,20 @@
 			"name": "White-out",
 			"icon": "tools/pencil/whiteout_tape.svg",
 			"active": false,
-			"switch": stopLine,
+			"switch": function() {
+				stopLine();
+				toggleSize();
+			},
+		},
+		"onquit": function() {
+			//When switching to another tool, restore Pencil and its size
```

Comment:
Not sure though, if this would conflict with the way other people may use White-out?